lib/middleware/util.ts

```diff
@@ -21,7 +21,7 @@ export function getCert(req: Interface.IOurRequest,
                         next: restify.Next): void
 {
     // Get client certificate details(only in https mode)
-    if (conf.get('https.enable')) {
+    if (req.isSecure()) {
         req.clientCert = (<tls.ClearTextStream>req.connection).getPeerCertificate();
```

Comment:
Has to change `restify` typing to expose this function. Depends on PR borisyankov/DefinitelyTyped#3626